src/main/java/org/mifos/connector/common/exception/PaymentHubErrorCode.java

```diff
@@ -0,0 +1,21 @@
+package org.mifos.connector.common.exception;
+
+public enum PaymentHubErrorCode {
```

Comment:
use paymenthub error category enum which is already available. this is duplicate code